frontend/src/styles.scss

```diff
@@ -264,3 +264,11 @@ body {
 .ant-select-dropdown {
 	z-index: 3000000000000 !important;
 }
+
+.ant-picker-dropdown {
+	z-index: 3000000000000 !important;
```

Comment:
Why are we setting such high numbers for z-index? They should be incremental.

We should have a place where we can look up for the layout hierarchy

Reply:
Because the default numbers are very high already for intercom and we needed to exceed that

Reply:
We are increasing them randomly without consideration. We should fix all these places.  The next similar PR will have an even higher number.

Reply:
Check if you can update the old values in this PR or let's create a PR to fix the old values and then raise this PR on top on that.

Reply:
Not increasing them randomly. The drawer is set to be 1*(base limit) . the modals need to be rendered above drawer which is (2*base limit ). then modals and drawers can have dropdowns and popovers which is 3*(base limit )

Reply:
drawer z-index is 1000

Reply:
```
.ant-drawer {
	// do not modify this as this is required to hide intercom in modals
	z-index: 1000000000000;
}

.ant-modal-root {
	.ant-modal-mask {
		z-index: 2000000000000 !important;
	}
	.ant-modal-wrap {
		z-index: 2000000000000 !important;
	}
}

.ant-select-dropdown {
	z-index: 3000000000000 !important;
}```

Reply:
<img width="262" alt="image" src="https://github.com/SigNoz/signoz/assets/54737045/6739b74c-5529-4d3f-ad2a-81133c47b91d">
This is the z-index for intercom

Reply:
> ```
> .ant-drawer {
> 	// do not modify this as this is required to hide intercom in modals
> 	z-index: 1000000000000;
> }
> 
> .ant-modal-root {
> 	.ant-modal-mask {
> 		z-index: 2000000000000 !important;
> 	}
> 	.ant-modal-wrap {
> 		z-index: 2000000000000 !important;
> 	}
> }
> 
> .ant-select-dropdown {
> 	z-index: 3000000000000 !important;
> }```
> ```

This was written by us.